frontend/src/components/Utils/filter-input.js

```diff
@@ -58,16 +59,19 @@ export function formatTripData(rawTripObj) {
   const defaultName = "Untitled Trip";
   const defaultDestination = "No Destination"
 
-  const formattedTripObj = {
-    trip_creation_time: firebase.firestore.Timestamp.now(),
-    name:               getCleanedTextInput(rawTripObj.name, defaultName),
-    description:        rawTripObj.description,
-    destination:        getCleanedTextInput(rawTripObj.destination,
-                                                 defaultDestination),
-    start_date:         getTimestampFromDateString(rawTripObj.startDate),
-    end_date:           getTimestampFromDateString(rawTripObj.endDate),
-    collaborators:      getCollaboratorUidArray(rawTripObj.collaboratorEmails),
-  };
+  const tripObj = {};
+  tripObj[DB.TRIPS_UPDATE_TIMESTAMP] = firebase.firestore.Timestamp.now();
+  tripObj[DB.TRIPS_TITLE] =
+      getCleanedTextInput(rawTripObj[DB.TRIPS_TITLE], defaultName);
+  tripObj[DB.TRIPS_DESCRIPTION] = rawTripObj[DB.TRIPS_DESCRIPTION];
+  tripObj[DB.TRIPS_DESTINATION] =
+      getCleanedTextInput(rawTripObj[DB.TRIPS_DESTINATION], defaultDestination);
+  tripObj[DB.TRIPS_START_DATE] =
+      getTimestampFromDateString(rawTripObj[DB.TRIPS_START_DATE]);
+  tripObj[DB.TRIPS_END_DATE] =
+      getTimestampFromDateString(rawTripObj[DB.TRIPS_END_DATE]);
+  tripObj[DB.TRIPS_COLLABORATORS] =
+      getCollaboratorUidArray(rawTripObj[DB.TRIPS_COLLABORATORS]);
```

Comment:
What's the difference between this and just replacing strings like "trip_creation_time" with "DB.TRIPS_UPDATE_TIMESTAMP"?

Reply:
After chatting some offline, found that this does not work in ES5 or earlier but there is a way to do it in ES6 with `ComputedPropertyName` syntax (learned about it [here](https://stackoverflow.com/questions/2274242/how-to-use-a-variable-for-a-key-in-a-javascript-object-literal)). Changed to use this.